### ISSUE TITLE
points can't be printed out on a postscript file (save as)

### ISSUE THREAD

**Issue by lixun910** [GeoDaCenter/geoda#634]
this is what should looks like when print out points to ps file
<img width="914" alt="screen shot 2017-03-02 at 5 24 43 pm" src="https://cloud.githubusercontent.com/assets/2423887/23531702/4ccffebc-ff6d-11e6-9128-0af56a27e1ad.png">

**Comment by lanselin** [comment]
I think there is a larger problem here, namely the use of the empty circles (only the edge is shown) to depict points. It is very hard to see different colors (on a choropleth map) or the categories in a LISA cluster map. We may want to think about a better solution, such as full circles (or squares). I remember this was initially done because of drawing performance reasons, but maybe that is no longer relevant. It would make the point maps much more useful if you could actually see the colors!

**Comment by lixun910** [comment]
Yes there was a performance issue, especially when supporting real transparency. I will add some customized options in GeoDa so that user can choose either better performance or "better looking".

**Comment by luluzhou766** [comment]
Window 7
Geoda 1.12.1.103
There is no PS file option in Save As.
![7 mar 04](https://user-images.githubusercontent.com/36253697/37112183-a89f214c-2239-11e8-95ff-b90c256b9a45.JPG)

The points are still empty circles. But you do have choice to change color in choropleth map and LISA cluster map. 


![7 mar 05](https://user-images.githubusercontent.com/36253697/37112340-01f0a4d2-223a-11e8-96fd-88545917175a.JPG)

The other problems is the legends and points overlapping when save the image. 
![7 mar 02](https://user-images.githubusercontent.com/36253697/37112409-3a5d49b0-223a-11e8-954a-a2f352ed0e40.png)

**Comment by jkoschinsky** [comment]
Right-click on map and saving image as postscript does work:

<img width="1228" alt="screen shot 2018-03-08 at 5 28 37 pm" src="https://user-images.githubusercontent.com/13679743/37182295-3116b640-22f6-11e8-8cd8-884acb9834f7.png">

<img width="1629" alt="screen shot 2018-03-08 at 5 31 19 pm" src="https://user-images.githubusercontent.com/13679743/37182384-9d981d0e-22f6-11e8-8f2c-7c003e00022b.png">

but checking or unchecking Color > Outlines Visible doesn't do anything:

<img width="598" alt="screen shot 2018-03-08 at 5 35 07 pm" src="https://user-images.githubusercontent.com/13679743/37182483-2f05430c-22f7-11e8-922a-da0700fbbcc5.png">
